src/ControlResolver.as

```diff
@@ -176,32 +211,32 @@ package {
                 "USB Gamepad": {  // USB NES controller
                     "Win": {
                         "right": {
-                            "button": "AXIS_3",
+                            "button": "axis_0",
                             "value_on": 1,
                             "value_off": 0
                         },
                         "left": {
-                            "button": "AXIS_3",
+                            "button": "axis_0",
                             "value_on": -1,
```

Comment:
These values need to be changed to match the raw axis values from the controller driver